tests/liteclient_test.cc

```diff
@@ -385,9 +385,11 @@ class LiteClientTest : public ::testing::Test {
   /**
    * method createApp
    */
-  AppEngine::App createApp(const std::string& name, const std::string& factory = "test-factory") {
-    const std::string uri = "localhost:" + getDeviceGateway().getPort() + "/" + factory + "/" + name +
-                            "@sha256:7ca42b1567ca068dfd6a5392432a5a36700a4aa3e321922e91d974f832a2f243";
+  AppEngine::App createApp(const std::string& name,
```

Comment:
I think we should modify the params since once we switch to using the http client registry we will only need the factory and the name...so maybe having the order as name, factory and hash so we can keep a call with two defaults